authentication/authentication-styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+}
```

Comment:
consider using short form of this
```
margin: 100px 20px 50px 20px;
```

Consider using padding here instead of margin for body. We want the body itself to cover the whole screen. Margin will left the screen spaces